scripts/ci/jobs/push-images.sh

```diff
@@ -44,6 +44,10 @@ push_images() {
         push_context="merge-to-master"
     fi
 
+    # TODO(ROX-11889): make this unconditional once the openshift/release side is ready
```

Comment:
nit: shouldn't operator be after main

Reply:
I explain the reason for this order in the commit description.

>  I assume that this passed in the rehearsal PR somewhere and a link to a log would be useful.

As mentioned in the PR description:
> Roughly the same changes (module some debug output and comments) were tested in https://github.com/openshift/release/pull/30630

I avoided doing this for this PR, because it would cause the operator images to be built both by prow and by circleci and clash. OTOH if I made a change here to remove the CircleCI operator jobs in this PR, then we wouldn't know if this PR didn't break them. But I guess that's OK since they are scheduled for disabling today.